pkg/kfapp/ossm/types.go

```diff
@@ -28,6 +25,14 @@ type manifest struct {
 	processed bool
 }
 
+// In order to process the templates, we need to create a tmp directory
+// to store the files. This is because embedded files are read only.
+const (
```

Comment:
```suggestion
const (
```

Not sure this belongs here.